setup.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    remove: [
+        { file: 'resources/icons/512x512.png' },
+        { file: 'resources/osx/dmg-background.png' },
+        { file: 'resources/osx/dmg-background@2x.png' },
+        { file: 'resources/osx/dmg-icon.icns' },
+        { file: 'resources/osx/icon.icns' },
+        { file: 'resources/windows/icon.ico' },
+        { file: 'resources/windows/setup-icon.ico' },
+        { folder: 'src/hello_world' }
```

Comment:
I was suggesting you should add `setup.js` to the list of files to delete, since you'll never need it again.